tox.ini

```diff
@@ -36,6 +36,7 @@ deps =
     pytest-xdist<1.28.0  # pyup: < 1.28 # blocked by `pytest`
     shade==1.31.0
     yapf>=0.25.0,<0.27  # pyup: < 0.27 # disable updates, conflicts with flake8 as per https://github.com/ansible/molecule/pull/1915
+    selinux  # https://github.com/ansible/molecule/issues/1724
```

Comment:
This linked issue doesn't explain anything. It's misleading to me. We need some actual comment here.